### ISSUE TITLE
STPPaymentCardTextField- UIAppearance proxies being overridden

### ISSUE THREAD

**Issue by markcleonard** [stripe/stripe-ios#482]
## Summary

Setting the appearance proxies for `borderColor` and `borderWidth` on the `STPPaymentCardTextField` is being overridden by the default vales during initialization. 

## iOS version

9.2

## Installation method

Cocoapods

## SDK version

9.0.0

## Other information

```
    self.borderColor = [self.class placeholderGrayColor];
    self.cornerRadius = 5.0f;
    self.borderWidth = 1.0f;
```

These are ignoring any value set by the appearance proxy.

**Comment by markcleonard** [comment]
Additionally, setting a borderColor to nil does not seem to remove the border, as is indicated in the comment above the property. Instead, a black border is drawn.

**Comment by benzguo** [comment]
Hey @markcleonard , thanks for reporting this issue! We've merged a fix to master (#483) – it'll be out in the next release.
